tests/update/patch_sle.pm

```diff
@@ -124,6 +127,13 @@ sub install_packages {
     }
 }
 
+# Install packages salt-master salt-minion before migration, to ensure salt
+# regression test work well even the asmm is disabled after migration.
+sub install_salt_packages {
+    zypper_call "in -t package 'salt-master'";
+    zypper_call "in -t package 'salt-minion'";
```

Comment:
Why not one line `zypper_call('in salt-master salt-minion');` :)

Reply:
Updated, thanks.